src/js/common/stories/HowItWorksWizzard.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import HowItWorks from '../../components/CompleteYourProfile/CompleteYourProfile2024';
+
+export default {
+  title: 'Design System/Wizzard',
+  component: HowItWorks,
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+`;
+
+export const HowItWorksWizzard = () => (
```

Comment:
"Wizard" is spelled with one Z. I am approving this pull request, but could you make this change in a future pull request? Thank you!